cw-plus/cw20/advanced.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Advanced Usage
-order: 3
+order: 4
```

Comment:
Maybe this belongs in "Overview" or "General" as these tips are valid for all contracts

Reply:
Nice idea.